sessionctx/variable/sysvar.go

```diff
@@ -581,3 +581,27 @@ type GlobalSysVarAccessor interface {
 	// SetGlobalSysVar sets the global system variable name to value.
 	SetGlobalSysVar(ctx context.Context, name string, value string) error
```

Comment:
The struct name is GlobalSysVarAccessor which indicates that we will handle global sys var, so I thin function Set/Get is enough.